nerdtribe/mr-binge
# Media Details Stubbing

## Explanation of work done and why:


Create a stubbed out component for supporting media details page.
-  Details Page
-  Routing
-  Pass id in route

## How to (manually) test:

1. Fetch `git fetch && git checkout -b media-details origin/media-details`
2. Serve `yarn electron:serve`
3. Observe
- Clicking on a _Movie Card_ navigates you to the detail page with a unique id passed in.
- Clicking on a TV Card_ navigates you to the detail page with a unique id passed in.
![image](https://user-images.githubusercontent.com/8315447/68036564-543ea480-fc9c-11e9-85ef-f6edfe70bd41.png)

## Checklist:
Closes #37
